config/master/aws-k8s-cni.yaml

```diff
@@ -123,27 +123,26 @@ spec:
       priorityClassName: "system-node-critical"
       serviceAccountName: aws-node
       hostNetwork: true
-      initContainers:
-      - name: aws-vpc-cni-init
-        image: "602401143452.dkr.ecr.us-west-2.amazonaws.com/amazon-k8s-cni-init:v1.12.0"
-        env:
-          - name: DISABLE_TCP_EARLY_DEMUX
-            value: "false"
-          - name: ENABLE_IPv6
-            value: "false"
-        securityContext:
+      containers:
+        - name: aws-vpc-cni-init
+          image: "602401143452.dkr.ecr.us-west-2.amazonaws.com/amazon-k8s-cni-init:v1.12.0-11-g69122330"
```

Comment:
@jayanthvn what should the tag be set to when making a change like this? If we keep it at `v1.12.0`, then the manifest and image are incompatible, but I doubt we want to release a new tag. Do we guarantee that anyone cloning this should be able to run `kubectl apply -f config/master/aws-k8s-cni.yaml`?

Reply:
Yeah we will have build a RC image and use that tag here since both manifest and the entry point is changing.